### DISCUSSION TITLE
Backups are not saved under the specified disk.

### DISCUSSION THREAD

**Discussion by fikoulli** [spatie/laravel-backup#1492] [Q&A]
Hi there,

First, Thx for this package,

I just installed laravel-backup but I have one small issue : 
     the backups are not saved in the specified driver but under /storage/app/[app-name]/

Here's the config/backup.php

    'name' => env('APP_NAME', 'laravel-backup'),
    'destination' => [
        'filename_prefix' => '',
        'disks' => [
                'backups',
            ],
        ],
            
And  config/filesystems.php    
    
    'disks' => [
        'backups' => [
            'driver' => 'local',
            'root' => '/var/www/html/backups',
            'visibility' => 'public',
        ],

And .env, something like :

    APP_NAME="Hello World"

Versions : 
php : 7.4.28
laravel/framework :  7.30.4
spatie/laravel-backup : 6.16.5

**Comment by PaolaRuby** [comment]
> spatie/laravel-backup : 6.16.5

laravel-backup v6 has no support, but did you try to clear config cache?

**Reply by fikoulli** [reply]
Thx for ur reply,

Yes, I cleared all what can be cleared, maybe i'll fix this with a symlink.

No support for V6 ? Thx for the info, So I should close this issue.
